scripts/Database-Update.ps1

```diff
@@ -1,6 +1,6 @@
 Push-Location -Path $PSScriptRoot\..\backend\YouTubeV2.Application
 
 dotnet ef database update --startup-project ..\YouTubeV2
-dotnet ef database update --connection "Server=localhost;Integrated Security = true;Database=YouTubeV2Test;TrustServerCertificate=True" --startup-project ..\YouTubeV2
+dotnet ef database update --connection "Server=localhost;User Id=sa;Password=4C0b138f;Database=YouTubeV2Test;TrustServerCertificate=True" --startup-project ..\YouTubeV2
```

Comment:
I dont think we should change this

Reply:
That's actually a mistake, gonna fix this